tests/imap/test_save_folder_names.py

```diff
@@ -19,7 +19,7 @@ def test_imap_save_generic_folder_names(db, default_account):
     raw_folders = [RawFolder(*args) for args in folder_names_and_roles]
     monitor.save_folder_names(db.session, raw_folders)
     saved_folder_data = set(
-        db.session.query(Folder.name, Folder.canonical_name).filter(
+        db.session.query(Folder.name, Folder._canonical_name).filter(
```

Comment:
Okay, then this looks good. Hopefully nothing else breaks later on. Thanks for fixing it!